### ISSUE TITLE
Performance decrease

### ISSUE THREAD

**Issue by chuck2098** [aryzhov/flutter-expandable#83]
Hi,
I observed that, when I use this plug-in for item of a listview with a lot of elements, than the performance of my app decreased significantly.
Have you observed this you too?

Thanks

**Comment by hicnar** [comment]
Yes. Using ListView.builder + if possible preloading all data presented on expandable panels helps
